front-end/app/components/map-comp.hbs

```diff
@@ -5,9 +5,14 @@
   @onClick={{this.onMapClick}}
   as |layers|
 >
+{{#if this.view}}
   <layers.tile
     @url='https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png'
   />
+  {{else}}
+  <layers.tile @url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"/>
```

Comment:
I think it would be a good idea to replace the base map with the more detailed map from OSM Belgium.
```suggestion
  <layers.tile @url="https://tile.openstreetmap.be/osmbe/{z}/{x}/{y}.png"/>
```